app/scripts/modules/core/src/notification/selector/types/pubsub/PubsubNotificationType.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { FormikFormField, TextInput } from 'core/presentation';
+import { INotificationTypeCustomConfig } from 'core/domain';
+
+export class PubsubNotificationType extends React.Component<INotificationTypeCustomConfig> {
+  constructor(props: INotificationTypeCustomConfig) {
```

Comment:
can probably remove this constructor as well, looks like this component could even be a SFC if ya want

Reply:
removed the constructor.

I followed the same pattern as how triggers were registered which set a `componentType` as part of its configuration.